### PR TITLE
add test for the external authorization task

### PR DESCRIPTION
**Pull request by yangminzhu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[x] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/security/authorization/authz-custom/test.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+kubectl get cm istio -n istio-system -o yaml | tr '\n' '^' | sed -e "s/${meshConfigPlaceholder}/${extensionProviders}/"  | tr '^' '\n' | kubectl apply -n istio-system -f -
+
+# restart istiod.
+snip_define_the_external_authorizer_4
+_wait_for_deployment istio-system istiod
+
+# deploy ext-authz service.
+snip_deploy_the_external_authorizer_1
+_wait_for_deployment foo ext-authz
+
```
Comment (ericvn) [inline content/en/docs/tasks/security/authorization/authz-custom/test.sh:47]: You might want to add a check of the logs:

Maybe something like:
```
_verify_lines snip_deploy_the_external_authorizer_2_out "
+ Starting gRPC server at
"
```
I'm not sure what to specifically look for in the log.

File: content/en/docs/tasks/security/authorization/authz-custom/test.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# create the authorization policy and verify the ext-authz response.
+snip_enable_with_external_authorization_1
+
+_verify_same snip_enable_with_external_authorization_2 "$snip_enable_with_external_authorization_2_out"
+_verify_same snip_enable_with_external_authorization_3 "$snip_enable_with_external_authorization_3_out"
+_verify_same snip_enable_with_external_authorization_4 "$snip_enable_with_external_authorization_4_out"
+
```
Comment (ericvn) [inline content/en/docs/tasks/security/authorization/authz-custom/test.sh:54]: Similar to above, add a check of the logs, this time looking for `[gRPCv2][allowed]` and `[gRPCv2][denied]`.

**Review by ericvn — COMMENTED** [review]
LGTM. Just a few comments on adding some `verifies` for the `kubectl logs` instructions on the page.

File: content/en/docs/tasks/security/authorization/authz-custom/test.sh (hunk 1 above)
Comment (yangminzhu) [reply content/en/docs/tasks/security/authorization/authz-custom/test.sh:47]: ah, thanks, the `_verify_lines` is very handy in this case.

**Review by ericvn — APPROVED** [review]
LGTM after adding the log checks.

**Pull request merged**